### PR TITLE
Core monthly 10/2023

### PR DESCRIPTION
**Pull request by kpodemski**
Core monthly for October.

### CODE REVIEW

**Review by tleon — APPROVED** [review]

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+date:   2023-11-06
+authors: [ kpodemski ]
+image: /assets/images/theme/banner-core-monthly.jpg
+twitter_image: /assets/images/theme/banner-core-monthly.jpg
+icon: icon-calendar
+tags:
+ - core-monthly
+---
+
+Check out what we've been up to in October with PrestaShop's core and modules, and get a sneak peek at the agenda and speakers for the upcoming PrestaShop Developer Conference.
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
In this Core Monthly, check out what we've been up to in October with PrestaShop's core and modules. Also, get a sneak peek at the agenda and speakers for the upcoming PrestaShop Developer Conference.
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+### PrestaShop Developer Conference
+
+A few weeks ago we closed the call for papers for the conference, and it's time to share with you the agenda and speakers for the event. We received a lot of great submissions, and we're excited to share the agenda with you.
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
A few weeks ago we closed the call for papers for the conference, and it's time to share with you the agenda as well as speakers for the event. We received a lot of great submissions, and we're excited to share the agenda with you.
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+It is evident that many of you have been eagerly anticipating the first ever PrestaShop Developer Conference. We're happy to see that the community is so engaged and willing to share their knowledge with others. We're confident that the PrestaShop Developer Conference is a great opportunity for you to learn new things and meet other developers.
+
+From many submissions we received we selected 17 talks that will be presented during the conference. We will be sharing more details about the agenda this week in the upcoming blog post, stay tuned!
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
From many submissions we received, we selected 17 talks that will be presented during the conference. We will be sharing more details about the agenda this week in a blog post, stay tuned!
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+
+The conference will take place on November 30, 2023 in Paris. You can find the agenda and speakers on the [conference website](https://events.prestashop.com/prestashop-developer-conference/en).
+
+### Changes on PrestaShop 8.1.x branch
+
+We are slowly getting closer to the next patch version for PrestaShop 8.1. We've recevied some feedback from the community that on the new product page performance with a lot of features is not optimal. We're still working on improving the performance of the page, and we will be sharing more details about the changes soon.
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
We are slowly getting closer to the next patch version for PrestaShop 8.1. We've received some feedback from the community: on the new product page, the performance, with a lot of features, is not optimal. We're still working on improving the performance of the page, and we will be sharing more details about the changes soon.
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+
+Other than that, there are some minor bugfixes which you can find in the list of pull requests below.
+
+### PrestaShop 9
+
+With PrestaShop 9 there's certainly more going on. Work on Symfony Layout, API, cleaning, a lot of progress, but the team still needs some time to present the results better. Regarding the new API which is being implemented into PrestaShop 9, [some interesting developments are happening](https://github.com/PrestaShop/PrestaShop/pull/34171), and we will be sharing more details about them soon!
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
With PrestaShop 9, there's certainly more going on. Work on Symfony Layout, API, cleaning: there is a lot of progress, but the team still needs some time to better present the results. Regarding the new API, which is being implemented into PrestaShop 9, [some interesting developments are happening](https://github.com/PrestaShop/PrestaShop/pull/34171), and we will be sharing more details about them soon!
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+Great occasion to learn more about the new API is the PrestaShop Developer Conference where [@joelievre](https://github.com/jolelievre) will be [presenting the new API](https://events.prestashop.com/prestashop-developer-conference/en/sessions?searchtext=api&page=1) to the audience.
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
A great time to learn more about the new API is the PrestaShop Developer Conference, where [@joelievre](https://github.com/jolelievre) will be [presenting the new API](https://events.prestashop.com/prestashop-developer-conference/en/sessions?searchtext=api&page=1) to the audience.
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 7]
```diff
@@ -0,0 +1,372 @@
...
+
+### Faceted search module
+
+It is safe to say that for the last few months [@Hlavtox](https://github.com/Hlavtox) have taken over the development of the faceted search module. He's been working on improving the module and fixing bugs. In October he released a new version of the module with yet another highly anticipated feature - the ability to filter by discounts, on sale and new products. You can get the new version of the module and see full list of changes on [the module release page](https://github.com/PrestaShop/ps_facetedsearch/releases/tag/v3.14.0).
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
It is safe to say that for the last few months [@Hlavtox](https://github.com/Hlavtox) has taken over the development of the faceted search module. He's been working on improving the module and fixing bugs. In October, he released a new version of the module with yet another highly anticipated feature - the ability to filter by discounts, on sale and new products. You can get the new version of the module and see a full list of changes on [the module release page](https://github.com/PrestaShop/ps_facetedsearch/releases/tag/v3.14.0).
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 8]
```diff
@@ -0,0 +1,372 @@
...
+
+### PrestaShop Flashlight
+
+The [PrestaShop Flashlight project](https://github.com/PrestaShop/prestashop-flashlight) that was presented during October's Live Update has been improved thanks to [@jokesterfr](https://github.com/jokesterfr). New features, examples of use and more. Want to learn more about PrestaShop Flashlight? [Check the demo from the Live Update](https://youtu.be/tNAfcQhVmVM?t=817).
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
The [PrestaShop Flashlight project](https://github.com/PrestaShop/prestashop-flashlight) that was presented during October's Live Update has been improved thanks to [@jokesterfr](https://github.com/jokesterfr). With new features, examples of use and more. Want to learn more about PrestaShop Flashlight? [Check the demo from the Live Update](https://youtu.be/tNAfcQhVmVM?t=817).
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 9]
```diff
@@ -0,0 +1,372 @@
...
+
+### Top Translators website
+
+It's not that often when we can share news about the Top Translators website, but this month, we have some great news to share. The website has been updated thanks to incredible work from [@davidglezz](https://github.com/davidglezz). David migrated the website's code from React and jQuery to Vue.js, reduced the size of the page, and more. You can check the new version of the website on [the Top Translators website](https://translators.prestashop-project.org/). It looks the same, but it works better, is easier to maintain, and is more performant! Thank you [@davidglezz](https://github.com/davidglezz)!
+
+### Developer documentation
+
+Work on the developer documentation never stops! We have [some great projects in the pipeline](https://github.com/PrestaShop/docs/pulls), and we're excited to share them with you soon. In the meantime, you can check the improved "List of hooks" page, where we've implemented some visual changes to make it easier to navigate the page and for documentation maintainers to add new hooks to the list.
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
Work on the developer documentation never stops! We have [some great projects in the pipeline](https://github.com/PrestaShop/docs/pulls), and we're excited to share them with you soon. In the meantime, you can check the improved "List of hooks" page, where we've implemented some visual changes to make it easier to navigate the page, and for documentation maintainers, to add new hooks to the list.
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 10]
```diff
@@ -0,0 +1,372 @@
...
+
+### Vat number module
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
### VAT number module
```

File: content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md [hunk 11]
```diff
@@ -0,0 +1,372 @@
...
+
+The `vatnumber` module that was archived in October 2023, is now maintained by [Friends of Presta association](https://friendsofpresta.org/). You'll now be able to download the new version of this module from [their `vatnumber` module repository](https://github.com/friends-of-presta/vatnumber).
```
Comment (coffrep) [inline content/news/core-monthly/2023/core-monthly-2023-10-01-2023-10-31.md]:
```suggestion
The `vatnumber` module that was archived in October 2023, is now maintained by the [Friends of Presta association](https://friendsofpresta.org/). You'll now be able to download the new version of this module from [their `vatnumber` module repository](https://github.com/friends-of-presta/vatnumber).
```

**Review by coffrep — APPROVED** [review]
Hi Krystian, I've made some suggestions, hope they make sense for you and do let me know of your thoughts. Also approving on my end.

**Pull request merged**